components/slack/sources/new-reaction-added/new-reaction-added.mjs

```diff
@@ -53,6 +53,12 @@ export default {
       type: "string[]",
       optional: true,
     },
+    includeUserData: {
+      description: "Include user object on the response. Default `false`",
```

Comment:
```suggestion
      description: "Include user object in the response. Default `false`"
```

Reply:
Changed "on" to "in" to follow Google's Developer docs style guide (https://developers.google.com/style/prepositions)